doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -199,6 +199,9 @@ Samples
 Bluetooth samples
 -----------------
 
+* :ref:`peripheral_hids_keyboard` and :ref:`peripheral_hids_mouse` samples register HID Service before Bluetooth is enabled (before calling the :c:func:`bt_enable` function).
+  The :c:func:`bt_gatt_service_register` function can no longer be called after enabling Bluetooth and before loading settings.
+
```

Comment:
```suggestion

* :ref:`peripheral_hids_keyboard` sample:

  * Updated the sample to register the HID Service before enabling Bluetooth (calling the :c:func:`bt_enable` function).
    The :c:func:`bt_gatt_service_register` function can no longer be called after enabling Bluetooth and before loading settings.
  
* :ref:`peripheral_hids_mouse` sample:

  * Updated the sample to register the HID Service before enabling Bluetooth (calling the :c:func:`bt_enable` function).
    The :c:func:`bt_gatt_service_register` function can no longer be called after enabling Bluetooth and before loading settings.
```

tbc - suggested 'updated' but please let me know if this is incorrect

Reply:
The commit was not in scope of my original PR. It was introduced here: https://github.com/jfischer-no/sdk-nrf/pull/4 and it was accepted by Divya, can we keep it ?